site/content/en/docs/commands/addons.md

```diff
@@ -128,6 +128,7 @@ minikube addons enable dashboard
 ### Options
 
 ```
+      --force               If true, will perform potentially dangerous operations. Use with discretion.
```

Comment:
`If true, force to perform possibly dangerous or not recommended operations at user's discretion`

Reply:
how is this better